host/sgx/switchless.c

```diff
@@ -10,6 +11,68 @@
 #include "../ocalls.h"
 #include "enclave.h"
 
+#if __linux__
+#include <linux/futex.h>
+#include <sys/syscall.h>
+#include <unistd.h>
+
+static void _worker_wait(oe_host_worker_context_t* context)
+{
+    if (__sync_val_compare_and_swap(&context->event, 1, 0) == 0)
+    {
+        do
+        {
+            syscall(
```

Comment:
Does syscall return an error in any scenario?

Reply:
Only if the input parameters are invalid, which we have ensured that they are not invalid. Besides it does not matter because we are looping until the event is non-zero.

Reply:
@mikbras Thanks for the explanation.
Added comment indicating that it is OK to ignore the syscall return values since we are looping until event is non-zero.